includes/core-changes/corefx/5.0/environment-osversion-returns-correct-version.md

```diff
@@ -0,0 +1,37 @@
+### Environment.OSVersion returns the correct operating system version
+
+<xref:System.Environment.OSVersion?displayProperty=nameWithType> returns the actual version of the operating system (OS) instead of, for example, the OS that's selected for application compatibility.
+
+#### Change description
+
+In previous .NET version, <xref:System.Environment.OSVersion?displayProperty=nameWithType> returns an OS version that may be incorrect when an application runs under Windows compatibility mode. For more information, see [GetVersionExA function remarks](/windows/win32/api/sysinfoapi/nf-sysinfoapi-getversionexa#remarks).
```

Comment:
The remarks section of the API reference says:

> The Environment.OSVersion property does not provide a reliable way to identify the exact operating system and its version. Therefore, we do not recommend that you use this method. Instead:
>
> - To identify the operating system platform, use the RuntimeInformation.IsOSPlatform method.
> - Avoid writing code that depends on a reported operating system version. Instead, check for the availability of the features that your application needs.

Should this be updated to clarify that it's no longer valid for .NET 5.0, and probably give a link to this breaking change doc?

Reply:
Thanks, I updated it here: https://github.com/dotnet/dotnet-api-docs/pull/4674. In general, I always update the API ref for breaking changes.